.github/workflows/ci.yml

```diff
@@ -27,16 +27,23 @@ jobs:
       with:
         path: ./src/github.com/linuxkit/linuxkit
 
+    - name: Set path
+      run: |
+            echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
```

Comment:
Looks like previously it was _appending_ `GOPATH` to `PATH`, correct? (`echo "::set-env name=PATH::$PATH:$(go env GOPATH)/bin"`)

Reply:
In case it's useful; here's how these are set in containerd/cri (but perhaps that CI differs); https://github.com/containerd/cri/blob/release/1.4/.github/workflows/ci.yml#L22-L23

Reply:
Yeah, it was appending `${GOPATH}/bin`, but it looks like we overrode `GOPATH` in the actions, which I missed in converting it. I think I got it now. I hope.

Reply:
Thanks for hopping in @thaJeztah 😄